foreign_cc/private/configure_script.bzl

```diff
@@ -70,7 +70,7 @@ def create_configure_script(
             options = " ".join(autoreconf_options),
         ).lstrip())
 
-    script.append("{env_vars} {prefix}\"{configure}\" --prefix=$$BUILD_TMPDIR$$/$$INSTALL_PREFIX$$ {user_options}".format(
+    script.append("{env_vars} {prefix}\"{configure}\" {user_options} --prefix=$$BUILD_TMPDIR$$/$$INSTALL_PREFIX$$".format(
```

Comment:
I wonder if we should provide an optional placeholder to allow the user to add `--prefix` into a specific place in the order of the arguments in case there is a configure script that isn't auto tools that expects it the other way around (or indeed in the middle of a list of options?)
Not every configure script is autotools based - which is the case with openssl (which is perl IIRC)